tests/TotalPriceCalculation.spec.ts

```diff
@@ -30,26 +23,63 @@ test('navigate to chairs page', async ({page})=>{
         await onLightingPage.scrollToProductElement()
 
         //click on the product
-        await onLightingPage.ClickOnAdjustableWallLampProduct();
+        await onLightingPage.clickOnAdjustableWallLampProduct();
+       
+        //Get the value of the first element.
+        const firstPrice = await page.locator('.price').innerText() 
+        console.log("FIRST PRICE:", firstPrice)
 
         //Click on the "Add to element" button
         await onLightingPage.clickOnAddToElementButton()
 
         //navigate to card page
-        await onLightingPage.navigateToCard()
+        await onLightingPage.navigateToCartPage()
+
+        //Get the value of the second element.
+        const priceInCartPage = await page.locator('.product-price').innerText()
```

Comment:
- Please, move locator to the Page class.
- Change name to something more descriptive